moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -141,7 +167,7 @@ const ComponentForm = ({
             startIcon={<CheckCircle />}
             type="submit"
           >
-            {formButtonText}
+            {isSignalComponent ? "Save Signal Component" : formButtonText}
```

Comment:
I don't feel strongly about the "Save Signal Component" text here. Maybe there is a better way to put this? I want it to communicate to users that they will be saving a signal component immediately when they click this button. 

Normally, the button says "Continue" because the next step is to define the geometry of the component with the map UI. We don't have to do that since there is already a geometry associated with the choice in the signal autocomplete dropdown.

Reply:
thanks for catching this! let's go with plain ole' "Save" since that's the text we show to users after they're done mapping.

Reply:
Perfect - I'll update that text to keep it simple. 🙏